protobuf/bess_msg.proto

```diff
@@ -151,6 +151,18 @@ message ListWorkersResponse {
   repeated WorkerStatus workers_status = 2;
 }
 
+message GetLinkStatusRequest {
```

Comment:
I believe the message name is self describing... Justine's comments are about module-specific messages (module_msg.proto), not system-wide messages.